provisioning/roles/utils/celery/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Ensure supervisor is installed and up-to-date.
```

Comment:
@Elarnon Generalising the `supervisor` file, you mean? I'm mixed about this. Let's discuss into another issue.